preproc/convert.py

```diff
@@ -172,10 +149,7 @@ def neuromod_phys2bids(sourcedata, scratch, sub, sessions=None):
         print("~" * 30)
 
 
-def _main(argv=None):
-    options = _get_parser().parse_args(argv)
-    neuromod_phys2bids(**vars(options))
-
-
 if __name__ == "__main__":
```

Comment:
here you can follow this template if you want to change the parser function for the click structure